src/aind_metadata_mapper/smartspim/utils.py

```diff
@@ -110,39 +138,18 @@ def make_acq_tiles(metadata_dict: dict, filter_mapping: dict):
         List with the metadata for the tiles
     """
 
-    channels = {}
-
     # List where the metadata of the acquired
     # tiles is stored
     tile_acquisitions = []
 
-    filter_wheel_idx = 0
-    for wavelength, value in metadata_dict["wavelength_config"].items():
-        channel = tile.Channel(
-            channel_name=str(wavelength),
-            light_source_name=str(wavelength),
-            filter_names=[""],  # We don't have filter names at the moment
-            detector_name="",  # We don't have detector names at the moment
-            additional_device_names=[],
-            # Excitation wavelengths
-            excitation_wavelength=int(wavelength),
-            excitation_wavelength_unit=SizeUnit.NM,
-            # Excitation power
-            excitation_power=value[
-                "power_left"
-            ],  # [value["power_left"], value["power_right"]]
-            excitation_power_unit=PowerUnit.PERCENT,
-            filter_wheel_index=filter_wheel_idx,
-        )
-        filter_wheel_idx += 1
-
-        channels[wavelength] = channel
+    # Wavelength config
+    wavelength_config = metadata_dict.get("wavelength_config")
 
     # Scale metadata
     session_config = metadata_dict.get("session_config")
 
-    x_res = y_res = session_config.get("µm/pix")
-    z_res = session_config.get("z_step_um")
+    x_res = y_res = session_config.get("um/pix")
+    z_res = session_config.get("Z step (um)")
```

Comment:
The key in session_config for "Z step" was different in Mike's script so I updated it here. Let me know if this shouldnt be changed